### DISCUSSION TITLE
Getting started help - edit page with Select examples

### DISCUSSION THREAD

**Discussion by dpyeates** [refinedev/refine#6579] [Q&A] [answered]
Hi there,

Just starting with Refine but have been using React for quite a while.

I am attempting to get a simple create/edit/list/show page working. I have the create and list working OK but I am really struggling with edit. I am using MUI, and am expecting the values of my edit form to be pre-populated with the values of the entry that I click on on my list when the edit page is displayed. but this is not happening.

Is there an example anywhere that clearly shows Refine/MUI/react-hook-form example using a Select? I've found examples with TextFields but anything more complex than this seems hard to find.

Thanks,
Darren.

**Comment by BatuhanW** [comment, marked as answer]
Hello @dpyeates we have `finefoods-material-ui` for more realistic app example. I think that can be more useful to you.

You can use following command to create on your local:

`npm create refine-app@latest -- --example finefoods-material-ui`

Also we have an AI Chatbot in our documentation, it should be able to help you for your specific questions.

https://refine.dev/docs
